src/libsystemd-network/sd-ipv4acd.c

```diff
@@ -442,6 +464,18 @@ int sd_ipv4acd_set_ifname(sd_ipv4acd *acd, const char *ifname) {
         return free_and_strdup(&acd->ifname, ifname);
 }
 
+int sd_ipv4acd_set_timeout(sd_ipv4acd *acd, uint64_t timeout_usec) {
+        assert_return(acd, -EINVAL);
+
+        if (timeout_usec == 0)
+                timeout_usec = TIMEOUT_DEFAULT_USEC;
+
+        /* Clamp the total duration to a value between 1ms and 1 minute */
+        acd->time_unit = CLAMP(timeout_usec, 1U * USEC_PER_MSEC, 60U * USEC_PER_SEC) / TOTAL_TIMEOUT;
```

Comment:
Please use `USEC_PER_MINUTE`.
Maybe better to use `DIV_ROUND_UP(..., TOTAL_TIMEOUT)`.